portfolio/src/main/webapp/comments.html

```diff
@@ -13,7 +13,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="fetchComments(true); fetchBlobstoreUrl();">
+  <body onload="loadCommentsPage();">
```

Comment:
I think you need to rename this to initiateCommentsPage() as well?

Reply:
omg I must have forgot